<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>
    <node  ns="/" name="stereo_image_proc" pkg="stereo_image_proc" type="stereo_image_proc" clear_params="true" output="screen">
       <remap from="/left/image_raw" to="/cam0/image_raw" />
       <remap from="/left/camera_info" to="/cam0/camera_info" />
       <remap from="/right/image_raw" to="/cam1/image_raw" />
       <remap from="/right/camera_info" to="/cam1/camera_info" />
        <param name="queue_size" value="200000" />
        <param name="disparity_range" value="128" />
    </node>
     <node  name="disp_view" pkg="image_view" type="disparity_view" clear_params="true" output="screen">
       <remap from="image" to="disparity" />
   </node> 
   <node  name="disp_error_view" pkg="image_view" type="disparity_view" clear_params="true" output="screen">
       <remap from="image" to="disparity_error" />
   </node>
   
   <node name="disparityErrorNode" pkg="disparity_error" type="disparityErrorNode" clear_params="true" output="screen">
        <param name="outputDirectoryPath" value="/home/hannes/Documents/Semesterprojekt/Experiments/move_70_2/" />
        <param name="x_crop_min" value="135" />
        <param name="x_crop_max" value="744" />
        <param name="y_crop_min" value="7" />
        <param name="y_crop_max" value="472" />
        <param name="plane_fitting_threshold" value="0.05" />
        <param name="enable_plane_fitting" value="false" />
   </node>
 

</launch>
